apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -174,6 +174,23 @@ defmodule OMG.Eth.RootChain do
     Eth.call_contract(contract, "hasToken(address)", [token], [:bool])
   end
 
+  def in_flight_exit(
+        in_flight_tx,
+        input_txs,
+        input_txs_inclusion_proofs,
+        in_flight_tx_sigs,
+        from,
+        bond,
```

Comment:
look at start_exti, bond is set in opts
 and default value is added to defaults